baseplate/clients/requests.py

```diff
@@ -204,13 +239,20 @@ def _add_span_context(self, span: Span, request: PreparedRequest) -> None:
 
     def send(self, request: PreparedRequest, **kwargs: Any) -> Response:
         """Send a :py:class:`~requests.PreparedRequest`."""
-        tags = {
-            "protocol": "http",
-            "http.method": request.method,
-            "http.url": request.url,
-            "http.slug": self.client_name if self.client_name is not None else self.name,
+        active_request_label_values = {
+            "http_method": request.method or "",
```

Comment:
to ensure consistency between libraries and http protocol versions can you please lowercase this?